config/email_service_provider.yml

```diff
@@ -0,0 +1,8 @@
+development:
+  provider: <%= ENV["GOVUK_EMAIL_ALERT_API_EMAIL_SERVICE_PROVIDER"] || "PSEUDO" %>
```

Comment:
Do we need the `GOVUK_EMAIL_ALERT_API_` prefix on these? We don't do that for other configuration values.

Reply:
Also not a fan of the all caps for the values, but that's just my personal preference.

Reply:
probably right. I was being overly pedantic with the naming.

Reply:
all caps feels to me more like a constant.....which these sort of represent. it doesn't make much difference to me though - if there's general outcry, I'm happy to lowercase them.